File: C++/674. Longest Continuous Increasing Subsequence.cpp
class Solution {
public:
    int findLengthOfLCIS(vector<int>& nums) {
        int sum = 0,ans = 0;
        int len = nums.size();
        for(int i=0;i<len;i++)
        {
            if(i==0 || nums[i]>nums[i-1])
                sum++;
            else
                sum = 1;
            ans = max(ans,sum);
        }
        return ans;
    }
};

File: Python/674. Longest Continuous Increasing Subsequence.py
class Solution:
    def findLengthOfLCIS(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        cnt, ans = 0, 0
        for i in range(len(nums)):
            if i == 0 or nums[i] > nums[i-1]:
                cnt += 1
            else:
                cnt = 1
            ans = max(ans, cnt)
        return ans